### PR TITLE
fix(curriclum): modified add images to your website

### LINKED ISSUE #38047: Challenge instructions are not precise.
**Describe your problem and how to reproduce it:**
I was helping a member on the forum with a challenge.
The instructions (link below) are to place an image **_after_** the main element, but a moderator told them to put it _inside_ where there are two empty lines. The mod suggested that the instructions be improved and I report it on Github.

Whether the code is placed inside or outside the main element results in a pass.

**Add a Link to the page with the problem:**
https://www.freecodecamp.org/learn/responsive-web-design/basic-html-and-html5/add-images-to-your-website

Here's the forum thread:
https://www.freecodecamp.org/forum/t/i-couldnt-get-through-uploading-an-image/340110/16

**Tell us about your browser and operating system:**
* Browser Name: 
* Browser Version: 
* Operating System: 
Irrelevant

**If possible, add a screenshot here (you can drag and drop, png, jpg, gif, etc. in this box):**

### PR DESCRIPTION
**Pull request by philipobiorah**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

 Makes modification suggested by RandellDawson  comments on issue #38047 

Closes #38047

### CODE REVIEW

File: curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/add-images-to-your-website.english.md [hunk 1]
```diff
@@ -22,10 +22,10 @@ Let's add an <code>alt</code> attribute to our <code>img</code> example above:
 ## Instructions
 <section id='instructions'>
 Let's try to add an image to our website:
-Insert an <code>img</code> tag, after the <code>main</code> element.
+Within the existing <code>main</code> element, insert an <code>img</code> element before the existing <code>p</code> elements..
 Now set the <code>src</code> attribute so that it points to this url:
 <code>https://bit.ly/fcc-relaxing-cat</code>
-Finally don't forget to give your image an <code>alt</code> text.
+Finally, don't forget to give your <code>img</code>element an <code>alt</code>  attribute with applicable text.
```
Comment (RandellDawson) [inline curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/add-images-to-your-website.english.md]: You are missing a space between the `</code>` and the word `element`
```suggestion
Finally, don't forget to give your <code>img</code> element an <code>alt</code>  attribute with applicable text.
```

**Review by RandellDawson — CHANGES_REQUESTED** [review]
@philipobiorah See my suggested change.

**Review by RandellDawson — APPROVED** [review]
I think this PR looks good.

**Comment by philipobiorah** [comment]
Thanks

File: curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/add-images-to-your-website.english.md [hunk 2]
```diff
@@ -22,10 +22,10 @@ Let's add an <code>alt</code> attribute to our <code>img</code> example above:
 ## Instructions
 <section id='instructions'>
 Let's try to add an image to our website:
-Insert an <code>img</code> tag, after the <code>main</code> element.
+Within the existing <code>main</code> element, insert an <code>img</code> element before the existing <code>p</code> elements..
```
Comment (Manish-Giri) [inline curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/add-images-to-your-website.english.md]:
```suggestion
Within the existing <code>main</code> element, insert an <code>img</code> element before the existing <code>p</code> elements.
```

**Review by Manish-Giri — CHANGES_REQUESTED** [review]
There's an extra period at the end of the sentence.

**Comment by raisedadead** [comment]
Hi @philipobiorah 

Thanks for your PR, and congrats on your first contribution to the codebase 🎉 !

I recommend you use a more meaningful and succinct title for the PR next time, for example like the one I updated this PR to.

Thanks and happy contributing!

**Review by raisedadead — APPROVED** [review]

**Pull request merged**